app/views/finders/_search_results.html.erb

```diff
@@ -5,7 +5,8 @@
 <div class="finder-results js-finder-results">
   <%= render "govuk_publishing_components/components/document_list", {
     items: local_assigns[:document_list_component_data],
-    remove_underline: true
+    remove_underline: true,
+    remove_top_border_from_first_child: result_set_presenter.has_sort_options
```

Comment:
Is there something where this is set to true for the 1st child?

Reply:
Not with the main search results page https://www.gov.uk/search/all?keywords=parliament&order=relevance since the top border is necessary to visually separate the sort options from the document list and should be false.

For any specialist finder pages (or finder pages with no sort options) it should be true.